### PR TITLE
generate references and links from yaml

### PR DESCRIPTION
**Pull request by davidcarlisle**

### CODE REVIEW

File: _data/references.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- number: 1
+  title: "Automated tagging of LATEX documents — what is possible today, in 2023? TUGboat, 44(2):262–266, 2023"
+  authors: U. Fischer and F. Mittelbach
+  url: "https://www.latex-project.org/publications/2023-UFi-FMi-TUG-tb137fischer-tagging23.pdf"
+
+- number: 2
+  title: "Enhancing LATEX to automatically produce tagged and accessible PDF. TUGboat, 45(1), 2024"
```
Comment (FrankMittelbach) [inline _data/references.yml:7]: I had fixed LATEX -> LaTeX in my version

File: _data/tagging-status.yml [hunk 2]
```diff
@@ -77,26 +78,28 @@
...
    status: partial-support
-   comments: See comments in [[1]](#ref1). The package itself can't be loaded right now. However, the tagging code supports (most of) the document-level  syntax of `enumitem`, so that one cae use, for example, `\begin{itemize}[...]` out of the box.
+   comments: The package itself can't be loaded right now. However, the tagging code supports (most of) the document-level  syntax of `enumitem`, so that one cae use, for example, `\begin{itemize}[...]` out of the box.
    tasks: Check which keys are not yet supported and what else is missing. 
    issues: [61]
    related-issues: [4]
    updated: 2024-07-05
-
+   references: [1]
+
```
Comment (FrankMittelbach) [inline _data/tagging-status.yml:92]: cosmetic detail, but I would put reference directly after comments

**Review by FrankMittelbach — APPROVED** [review]

File: _data/references.yml (hunk 1 above)
Comment (davidcarlisle) [reply _data/references.yml]: fixed in f969da8

File: _data/tagging-status.yml (hunk 2 above)
Comment (davidcarlisle) [reply _data/tagging-status.yml:92]: yes I added it there as it simplified conversion but I can move, I did think coming after updated looked a bit odd.

Comment (davidcarlisle) [reply _data/tagging-status.yml]: fixed at ae5a8f4

**Pull request merged**